sdk/communication/AzureCommunicationChat/Source/Signaling/TrouterEventUtil.swift

```diff
@@ -104,28 +104,28 @@ public enum TrouterEventUtil {
     ///   - chatEventId: The ChatEventId, determines the type of ChatEvent that should be created.
     ///   - request: The request payload.
     /// - Returns: A chat event model.
-    public static func create(chatEvent chatEventId: ChatEventId, from request: TrouterRequest) throws -> Any {
+    public static func create(chatEvent chatEventId: ChatEventId, from _: TrouterRequest) -> EventType {
```

Comment:
This signature doesn't make sense. The `TrouterRequest` isn't being used. It would be much simpler to have `EventType` be something more descriptive like `TrouterEvent` and have the enum itself have an initializer that accepts the request and parses it into the appropriate event.